core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/dot-ema-info-display/dot-ema-info-display.component.ts

```diff
@@ -0,0 +1,127 @@
+import { NgIf } from '@angular/common';
+import {
+    ChangeDetectionStrategy,
+    Component,
+    Input,
+    OnChanges,
+    inject,
+    signal
+} from '@angular/core';
+import { Router } from '@angular/router';
+
+import { ButtonModule } from 'primeng/button';
+
+import { DotExperiment } from '@dotcms/dotcms-models';
+import { DotMessagePipe } from '@dotcms/ui';
+
+import { EditEmaStore } from '../../../dot-ema-shell/store/dot-ema.store';
+import { EDITOR_MODE } from '../../../shared/enums';
+import { EditorData } from '../../../shared/models';
+import { getIsDefaultVariant } from '../../../utils';
+
+interface InfoOptions {
+    icon: string;
+    info: {
+        message: string;
+        args: string[];
+    };
+    action?: () => void;
+    actionIcon?: string;
+}
+
+@Component({
+    selector: 'dot-ema-info-display',
+    standalone: true,
+    imports: [NgIf, ButtonModule, DotMessagePipe],
+    templateUrl: './dot-ema-info-display.component.html',
+    styleUrls: ['./dot-ema-info-display.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class DotEmaInfoDisplayComponent implements OnChanges {
+    @Input() editorData: EditorData;
+    @Input() currentExperiment: DotExperiment;
+
+    protected options = signal<InfoOptions>(undefined);
+
+    protected readonly store = inject(EditEmaStore);
+    protected readonly router = inject(Router);
+
+    protected readonly editorMode = EDITOR_MODE;
+
+    ngOnChanges() {
```

Comment:
with the input signal, can replace the ngOnChanges to convert options to computed signal.